CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 Changes
 =======
 
+Version v0.3.1 (released 2018-02-16):
```

Comment:
I would go with 0.4.0 (it adds a significant amount of new features). Perhaps even just go with an proper v1.0.0 release since it's been running stable for quite some time.